src/common/components/cards/DictionarySearchBar.tsx

```diff
@@ -4,23 +4,16 @@ import TextField from 'material-ui/TextField';
 import * as React from 'react';
 
 interface DictionarySearchBarProps {
-  onChange: (searchText: string) => void
-}
-
-interface DictionarySearchBarState {
   text: string
+  onChange: (searchText: string) => void
 }
 
-export default class DictionarySearchBar extends React.Component<DictionarySearchBarProps, DictionarySearchBarState> {
-  public state = {
-    text: ''
-  };
-
+export default class DictionarySearchBar extends React.Component<DictionarySearchBarProps> {
   public render(): JSX.Element {
     return (
       <div style={{display: 'flex', alignItems: 'center', width: '100%', height: 56}}>
         <TextField
-          value={this.state.text}
+          value={this.props.text}
```

Comment:
Might be worth taking the time to switch to the new TextField API?

Reply:
It's a TODO for sure.